examples/runners/wgpu/src/compute.rs

```diff
@@ -113,8 +113,9 @@ pub async fn start_internal(
         label: Some("Timestamps buffer"),
         size: 16,
         usage: wgpu::BufferUsage::MAP_READ | wgpu::BufferUsage::COPY_DST,
-        mapped_at_creation: false,
+        mapped_at_creation: true,
     });
+    timestamp_buffer.unmap();
```

Comment:
Why map and then immediately unmap?

Reply:
That's required to actually get values from the queries - there's a wgpu issue about it, which I can't find at the moment

Reply:
🤦

Reply:
https://github.com/gfx-rs/wgpu/issues/1374#issuecomment-839359307